recipes/ont-tombo/meta.yaml

```diff
@@ -15,39 +15,37 @@ build:
   entry_points:
     - tombo = tombo.__main__:main
   number: 0
-  skip: True  # [not py27]
 
 requirements:
   build:
     - python
     - setuptools
     - h5py
-    - rpy2 >=2.4.2
+    - rpy2
     - numpy
     - scipy
     - cython
     - r-base
     - r-ggplot2
-    - r-cowplot
-    - hdf5
-    - graphmap
-    - bwa
-    - minimap2
+    - r-gridextra
+    - mappy
+    - pyfaidx
+    - future
 
   run:
     - python
+    - setuptools
     - h5py
-    - rpy2 >=2.4.2
+    - rpy2
```

Comment:
Given https://github.com/nanoporetech/tombo/blob/1.2/setup.py#L37-L40, this should be
```yaml
    - rpy2  # [py3k]
    - rpy2 <=2.8.6  # [not py3k]
```